base/standard/src/main/java/org/openscience/cdk/aromaticity/AromaticType.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (C) 2024 John Mayfield
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ */
+
+package org.openscience.cdk.aromaticity;
+
+/**
+ * Aromatic atom types organized by element and degree, for example
+ * B2 => "Boron Degree 2" <code>SMILES: *B=*</code>.
```

Comment:
Where does this list come from? Is there any source that can be cited here?

Reply:
Possibly the OpenSmiles documentation (I did a table there) but this is a superset of that, it's more just evolved over time.